src/lib/questions.ts

```diff
@@ -535,19 +632,26 @@ export const questionsAndText: (
 				name: "authorName",
 				message: "Please enter your name (or nickname):",
 				action: checkAuthorName,
+				import: (ctx) => ctx.packageJson.author?.name,
 			},
 			{
 				type: "input",
 				name: "authorGithub",
 				message: "What's your name/org on GitHub?",
 				initial: ((answers: Answers) => answers.authorName) as any,
 				action: checkAuthorName,
+				import: (ctx) =>
+					ctx.ioPackageJson.common?.extIcon?.replace(
+						/^.+?\.com\/([^\/]+)\/.+$/,
+						"$1",
+					),
```

Comment:
I'd look at package.json -> author.name instead

Reply:
I thought so too, but some old projects might not have this in package.json. In addition, for adapters inside an org, that will not work. Example: https://github.com/iobroker-community-adapters/ioBroker.squeezebox/blob/5c7b6aa9181393d57cf13fa1d4d39c1f17904b7b/package.json#L5-L7
According to the adapter tester, `common.extIcon` must match the right repo.